### DISCUSSION TITLE
Running the development Engine version with IDE

### DISCUSSION THREAD

**Discussion by 4e6** [enso-org/enso#6234] [Work In Progress - Engine] [answered]
Running the development Engine version requires additional configuration to be done https://github.com/enso-org/enso/blob/37d820c764fd98b56990fb01c73502f49ad74431/docs/CONTRIBUTING.md#running-ide

> Copy or symlink the development version of the engine created with SBT's buildEnginedistribution command to the engines directory of the Enso distribution folder.

```
$ ls -l ~/.local/share/enso/dist/
lrwxrwxrwx 1 dbushev dbushev 98 Dec 14 16:50 2023.1.1-dev -> ../../../../projects/luna/enso/built-distribution/enso-engine-0.0.0-dev-linux-amd64/enso-0.0.0-dev/
```

where the location the `dist` directory can be found in the [distribution.md](https://github.com/enso-org/enso/blob/37d820c764fd98b56990fb01c73502f49ad74431/docs/distribution/distribution.md) doc, i.e. on macOS it will be `$HOME/Library/Application Support/org.enso/dist`.

Otherwise, the project manager will try to look up the `2023.1.1-dev` version in GitHub releases and fail when unable to find one.

**Comment by GregoryTravis** [comment, marked as answer]
Thanks!

I had to add a second link before it started working. This is what worked for me:

```
cd "/Users/gmt/Library/Application Support/org.enso/dist"
ln -s /Users/gmt/dev/enso/enso/built-distribution/enso-engine-0.0.0-dev-macos-amd64/enso-0.0.0-dev 2023.1.1-dev
ln -s /Users/gmt/dev/enso/enso/built-distribution/enso-engine-0.0.0-dev-macos-amd64/enso-0.0.0-dev 0.0.0-dev
```
